src/task.rs

```diff
@@ -2216,6 +2303,12 @@ impl TaskMetrics {
     pub fn mean_slow_poll_duration(&self) -> Duration {
         mean(self.total_slow_poll_duration, self.total_slow_poll_count)
     }
+
+    /// The average scheduling delay for a task which takes a long time to start executing after
+    /// being scheduled.
```

Comment:
Can you follow the documentation pattern of [`mean_slow_poll_duration`](https://docs.rs/tokio-metrics/0.2.0/tokio_metrics/struct.TaskMetrics.html#method.mean_slow_poll_duration) here?